src/useForm.tsx

```diff
@@ -177,13 +181,12 @@ function runValidation<T>(
   })
 }
 
-function maybeGetFirstValidationError<T, S>(
+function maybeGetFirstValidationError<T, U>(
```

Comment:
Conventionally we use `T`, `U`, `V` etc for type params, so I made that change here.